File: arrays-3.py
# https://leetcode.com/problems/remove-duplicates-from-sorted-array
def removeDuplicates(self, nums: List[int]) -> int:
        k = 0
        for i in range(len(nums)):
            if(nums[k] == nums[i]):
                pass
            else:
                nums[k+1] = nums[i]
                k += 1
        return k+1

File: arrays-3.c
# https://leetcode.com/problems/sort-colors/
int removeDuplicates(int* nums, int numsSize){
    if(numsSize==0)
        return 0;
    int k = 0;
    for (int i = 1; i < numsSize; i++){
        if (nums[i] != nums[k]){
            nums[k + 1] = nums[i];
            k++;
            }
        }
    return k + 1;
}
